config/routes.rb

```diff
@@ -1,5 +1,9 @@
 SolidErrors::Engine.routes.draw do
   get "/" => "errors#index", :as => :root
 
-  resources :errors, only: [:index, :show, :update], path: ""
+  resources :errors, only: [:index, :show, :update, :destroy], path: "" do
+    collection do
+      get :resolved
+    end
+  end
```

Comment:
When we collapse both views into the index route, we won't need this custom route.